PicturesVIP/PicturesVIP/DetailModule/DetailViewController.swift

```diff
@@ -0,0 +1,107 @@
+// DetailViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран модуля редактирования детальной информации по картинке
+final class DetailViewController: UIViewController, DetailDisplayLogic {
+    // MARK: - Visual Components
+    
+    lazy var textView: UITextView = {
+        let textView = UITextView()
+        textView.frame = CGRect(origin: .init(x: 0, y: 120), size: view.bounds.size)
+        textView.font = UIFont(name: S.fontArial, size: 15)
+        return textView
+    }()
+    
+    lazy var changeTextColor: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 10, y: 70, width: 160, height: 40)
+        button.backgroundColor = .darkGray
+        button.layer.cornerRadius = 5
+        button.clipsToBounds = true
+        button.tag = 0
+        button.setTitle(S.changeColorString, for: .normal)
+        button.addTarget(self, action: #selector(tapButtonAction), for: .touchUpInside)
+        return button
+    }()
+    
+    lazy var actuateDeepLink: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: view.bounds.width - 90, y: 20, width: 50, height: 50)
+        button.center.y = changeTextColor.center.y
+        button.layer.cornerRadius = 5
+        button.clipsToBounds = true
+        button.tag = 1
+        button.setBackgroundImage(UIImage(named: S.facebookIconString), for: .normal)
+        button.addTarget(self, action: #selector(tapButtonAction), for: .touchUpInside)
+        return button
+    }()
+    
+    // MARK: - Public Properties
+    
+    var interactor: DetailBisinessLogic?
+    var router: DetailRoutingLogic?
+    var isCased = false
+    
+    // MARK: - Initializers
+    
+    init(interactor: DetailBisinessLogic, router: DetailRoutingLogic) {
+        super.init(nibName: nil, bundle: nil)
+        self.interactor = interactor
+        self.router = router
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError(S.fatalErrorString)
+    }
+
+    // MARK: - lifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
```

Comment:
```suggestion
 initForm()
```
Тут  initForm(), а в нем вызов итерактора